src/button.component.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+import { Button as grommetButton, ButtonProps as grommetButtonProps, DropButton as grommetDropButton } from 'grommet'
+
+import { getColor, getDefaultTransition } from './provider.component'
+
+export interface ButtonProps extends grommetButtonProps{
+  title?: string,
+  iconName: string,
+  size?: (0.8 | 1.0 | 1.1 |1.2 | 1.33 | 1.5 | 1.75 | 2.0 | 2.33 | 2.5 | 3.0 ),
```

Comment:
Do we need so many different sizes for our button?

Reply:
image only buttons can vary quite a bit, but I'll remove some.